components/cu_transform/src/broadcaster.rs

```diff
@@ -0,0 +1,277 @@
+use crate::transform::StampedTransform;
+use cu29::clock::CuTime;
+use cu29::prelude::*;
+use cu_spatial_payloads::Transform3D;
+use std::fmt::Debug;
+use std::marker::PhantomData;
+
+#[derive(Debug, Default, Clone)]
+pub struct TransformBroadcastPayload<
+    T: Copy + Debug + Default + bincode::enc::Encode + bincode::de::Decode<()> + 'static,
+> {
+    pub transforms: Vec<StampedTransform<T>>,
```

Comment:
This needs to be constant size otherwise you will kill the copperlist linear access.